src/tests/ftest/daos_test/suite.yaml

```diff
@@ -28,7 +28,7 @@ timeouts:
   test_daos_extend_simple: 3600
   test_daos_oid_allocator: 640
   test_daos_checksum: 500
-  test_daos_rebuild_ec: 7200
+  test_daos_rebuild_ec: 7500
```

Comment:
As a side note, this is already a 2 hour timeout for a PR test and we really need to go the other direction with this timeout. This test has one of the biggest impacts to CI wait time

Reply:
Yes, I know, but in the latest CI test, `rebuild_ec` got timeout at the last three test case `REBUILD46` without any other failures. I also noticed similar rebuild_ec timeout in another PR. That is why I increased the timeout. In fact, `rebuild_ec` tests are not always so slow. So I hope such changes will not affect most of the normal test cases.